### ISSUE TITLE
Offline Pyret

### ISSUE THREAD

**Issue by sorawee** [brownplt/pyret-lang#408]
This is just a feature request.

It seems that opening Pyret has very high overhead. Is it possible to cache files and states (html5 offline app?) so that everything is almost ready for use. Indeed, there should be a process to check with the server whether newer version is released, and if so, users can choose to update Pyret (which will take time).

**Comment by schanzer** [comment]
@jpolitz , is this still an issue?

**Comment by schanzer** [comment]
@jpolitz I see a purgatory-PR for this, but I also feel like anchor has subsumed this?

**Comment by schanzer** [comment]
@jpolitz given the progress with pyret-npm, I wonder if the moral equivalent of a solution has already landed?

**Comment by jpolitz** [comment]
Yeah this is done with `pyret-npm` and the offline canvas, charts, etc work. Thanks for flagging.
